fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java

```diff
@@ -252,6 +258,10 @@ protected int runCountQuery(String sql, Object... searchArgs)
         double dbCallDuration;
 
         try {
+            if (log.isLoggable(Level.FINE)) {
+                log.fine("Count SQL = " + sql + NEWLINE + "; searchArgs="
```

Comment:
is this what's mentioned on 217? it's logged here?

Reply:
shouldn't have been removed from old code